src/pokemon_icon.c

```diff
@@ -962,7 +963,10 @@ const u8 *const gMonIconTable[] =
     [SPECIES_UNOWN_Z] = gMonIcon_UnownZ,
     [SPECIES_UNOWN_EMARK] = gMonIcon_UnownExclamationMark,
     [SPECIES_UNOWN_QMARK] = gMonIcon_UnownQuestionMark,
-
+};
+const u8 *const gMonIconTableFemale[] =
```

Comment:
You should probably separate `gMonIconTable` from `gMonIconTableFemale` with an emptu line instead of putting them so close together.

Reply:
I thought I did xD
One sec

Reply:
Done